app/src/main/java/org/simple/clinic/scheduleappointment/ScheduleAppointmentUpdate.kt

```diff
@@ -35,6 +35,19 @@ class ScheduleAppointmentUpdate(
       is AppointmentScheduled -> next(model.doneButtonStateChanged(ButtonState.SAVED), CloseSheet)
       SchedulingSkipped -> dispatch(LoadPatientDefaulterStatus(model.patientUuid))
       is PatientDefaulterStatusLoaded -> scheduleAutomaticAppointment(event, model)
+      is TeleconsultRecordLoaded -> teleconsultRecordLoaded(model, event)
+    }
+  }
+
+  private fun teleconsultRecordLoaded(
+      model: ScheduleAppointmentModel,
+      event: TeleconsultRecordLoaded
+  ): Next<ScheduleAppointmentModel, ScheduleAppointmentEffect> {
+    val teleconsultRecord = event.teleconsultRecord
+    return if (teleconsultRecord != null) {
+      next(model.teleconsultRecordLoaded(teleconsultRecord))
```

Comment:
Since `teleconsultRecordLoaded` method accepts a `null`, we can avoid the `null` check and pass the `event.teleconsultRecord` directly to `model.teleconsultRecordLoaded`